### PR TITLE
Hi, I added some new link of Free tools to give the community have mo…

### PR DESCRIPTION
**Pull request by ttkdew**
…re options. Thanks for building this great thing

### Free SaaS Offering Submission

Thank you for contributing to this list. This list is for **SaaS**
services that offer a **free tier** to help developers evaluate and
build something that users can later use and get support for.

The focus of this list is quite broad but we try to keep things
limited to that which infrastructure developers, like DevOps Practitioners,
would find useful.

We do not anymore accept cPanel like PHP+MySQL hosting services.

### New Submission Check List

Please ensure your submission ticks all of these boxes:

 - [ ] This is Software as a Service not self hosted
 - [ ] It has a free tier not just a free trial
 - [ ] The submission mentions what is free
 - [ ] The submission is not already present in the list

### Thank You

This list is the result of more than a thousand people contributing
to make something useful, we appreciate your efforts.

### CODE REVIEW

**Review by FatGrizzly — CHANGES_REQUESTED** [review]
Thanks for contributing, this file uses markdown format.

Please fix the issues.

File: README.md [hunk 1]
```diff
@@ -541,14 +541,17 @@ Table of Contents
...
+  * [SOAP UI] (https://www.soapui.org/] - This open-source API testing tool is designed for REST and SOAP Web Services. Some vital features include automated functional, performance, regression, and security testing
   * [testingbot.com](https://testingbot.com/) — Selenium Browser and Device Testing, [free for Open Source](https://testingbot.com/open-source)
   * [Testspace.com](https://testspace.com/) - A Dashboard for publishing automated test results and a Framework for implementing manual tests as code using GitHub. The service is [free for Open Source](https://github.com/marketplace/testspace-com) accounts for 450 results per month.
   * [tesults.com](https://www.tesults.com) — Test results reporting and test case management. Integrates with popular test frameworks. Open Source software developers, individuals, educators, and small teams getting started can request discounted and free offerings beyond basic free project.
+  * [Taurus] (https://gettaurus.org) - User can write test script via a single YAML file in a text editor, then execute on Taurus with command in the terminal for API Testing
```
Comment (rachit1601) [inline README.md:554]: remove space in between the brackets

File: README.md [hunk 2]
```diff
@@ -541,14 +541,17 @@ Table of Contents
...
   * [percy.io](https://percy.io) - Add visual testing to any web app, static site, style guide, or component library.  Unlimited team members, Demo app and unlimited projects, 5,000 snapshots / month.
   * [saucelabs.com](https://saucelabs.com/) — Cross browser testing, Selenium testing and mobile testing, [free for Open Source](https://saucelabs.com/open-source)
   * [snippets.uilicious.com](https://snippets.uilicious.com) - It's like CodePen, but for cross browser testing. UI-licious lets write tests like user stories, and offers a free platform - UI-licious Snippets - that allows you to run unlimited number of tests on Chrome for free with no sign up requred, for up to 3 minutes per test run. Found a bug? You can simply copy the unique url to your test to show your devs exactly how to reproduce the bug.
+  * [SOAP UI] (https://www.soapui.org/] - This open-source API testing tool is designed for REST and SOAP Web Services. Some vital features include automated functional, performance, regression, and security testing
```
Comment (rachit1601) [inline README.md:550]: remove space in between the brackets

File: README.md [hunk 3]
```diff
@@ -541,14 +541,17 @@ Table of Contents
   * [everystep-automation.com](https://www.everystep-automation.com/) — Records and replays all steps made in a web browser and creates scripts,... free with fewer options
   * [Gremlin](https://www.gremlin.com/gremlin-free-software) — Gremlin's Chaos Engineering tools allow you to safely, securely, and simply inject failure into your systems to find weaknesses before they cause customer-facing issues. Gremlin Free provides access to Shutdown and CPU attacks on up to 5 hosts or containers.
   * [gridlastic.com](https://www.gridlastic.com/) — Selenium Grid testing with free plan up to 4 simultaneous selenium nodes/10 grid starts/4,000 test minutes/month
+  * [katalon.com] (https://katalon.com) - Provides a testing platform that can help teams of all sizes at different levels of testing maturity, including  Katalon Studio, TestOps (+ Visual Testing free),TestCloud and Katalon Recorder.
```
Comment (rachit1601) [inline README.md:544]: remove space in between the brackets

**Review by rachit1601 — CHANGES_REQUESTED** [review]

**Review by ttkdew — COMMENTED** [review]
Hi, I changed the format, thanks

File: README.md [hunk 4]
```diff
@@ -541,14 +541,17 @@ Table of Contents
...
   * [percy.io](https://percy.io) - Add visual testing to any web app, static site, style guide, or component library.  Unlimited team members, Demo app and unlimited projects, 5,000 snapshots / month.
   * [saucelabs.com](https://saucelabs.com/) — Cross browser testing, Selenium testing and mobile testing, [free for Open Source](https://saucelabs.com/open-source)
   * [snippets.uilicious.com](https://snippets.uilicious.com) - It's like CodePen, but for cross browser testing. UI-licious lets write tests like user stories, and offers a free platform - UI-licious Snippets - that allows you to run unlimited number of tests on Chrome for free with no sign up requred, for up to 3 minutes per test run. Found a bug? You can simply copy the unique url to your test to show your devs exactly how to reproduce the bug.
+  * [SOAP UI](https://www.soapui.org/] - This open-source API testing tool is designed for REST and SOAP Web Services. Some vital features include automated functional, performance, regression, and security testing
```
Comment (FatGrizzly) [inline README.md:550]: Please fix the ending bracket. Thanks

**Review by FatGrizzly — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (rachit1601) [inline README.md]: Firstly wrong format, and secondly I it just shows a 30 day trial

File: README.md [hunk 5]
```diff
@@ -541,14 +541,17 @@ Table of Contents
...
   * [testingbot.com](https://testingbot.com/) — Selenium Browser and Device Testing, [free for Open Source](https://testingbot.com/open-source)
   * [Testspace.com](https://testspace.com/) - A Dashboard for publishing automated test results and a Framework for implementing manual tests as code using GitHub. The service is [free for Open Source](https://github.com/marketplace/testspace-com) accounts for 450 results per month.
   * [tesults.com](https://www.tesults.com) — Test results reporting and test case management. Integrates with popular test frameworks. Open Source software developers, individuals, educators, and small teams getting started can request discounted and free offerings beyond basic free project.
+  * [Taurus](https://gettaurus.org) - User can write test script via a single YAML file in a text editor, then execute on Taurus with command in the terminal for API Testing
```
Comment (rachit1601) [inline README.md:554]: I don't think this is a Saas. Its just a tool

File: README.md (hunk 4 above)
Comment (ttkdew) [reply README.md]: Updated, thanks so much!

**Review by ttkdew — COMMENTED** [review]
Updated, thanks

File: README.md [hunk 6]
```diff
@@ -1554,6 +1557,7 @@ Table of Contents
   * [Dev.to](https://dev.to/) - Where programmers share ideas and help each other grow.
   * [Hashnode](https://hashnode.com/) — Hassle-free Blogging Software for Developers!.
   * [Medium](https://medium.com/) — Get smarter about what matters to you.
+  * [devRant](https://devrant.com/) - Share and bond over successes and frustrations with code, tech and life as a programmer.
```
Comment (rachit1601) [inline README.md:1560]: This doesn't seem to be a blogging platform, its more like reddit for developers.

**Review by rachit1601 — CHANGES_REQUESTED** [review]

File: README.md (hunk 6 above)
Comment (ttkdew) [reply README.md]: Thanks, I removed SOAP UI, devRant and Taurus.
I will come up with other SaaS in the next update (maybe next month). Thanks

**Review by ttkdew — COMMENTED** [review]
updated, removed some

File: README.md [hunk 7]
```diff
@@ -1682,6 +1683,7 @@ HTML Themes and UI Kits to create your applications faster!
   * [SmallDev.tools](https://smalldev.tools/) — A free tool for developers that allows you to Encode/Decode various formats, Minify HTML/CSS/Javascript, Beautify, Generate Fake/Testing real like dataset in JSON/CSV & multiple other formats and many more features. With a delightful interface.
   * [UseCSV by Layercode](https://layercode.com/usecsv) — Add CSV and Excel import to your web app in minutes. Give your users an enjoyable and robust data import experience. Get Started for Free without any credit card details, and start integrating UseCSV today. You can create unlimited Importers and upload files up to 100Mb.
   * [Buttons Generator](https://markodenic.com/tools/buttons-generator/) — 100+ buttons you can use in your project.
+  * [SEO Minion](https://seominion.com/) - Free SEO Tools for all SEO checks
```
Comment (ripienaar) [inline README.md:1686]: This is not a SaaS its a browser extension that appears to operate on its own

Comment (ttkdew) [reply README.md]: Checked, done. Many thanks

**Review by ttkdew — COMMENTED** [review]
Updated

**Review by FatGrizzly — APPROVED** [review]

**Pull request merged**